ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -410,7 +410,8 @@ public static void justificationAndFinalization(
    * @param state
    * @param previous_total_balance
    */
-  public static void attestionInclusion(BeaconState state, UnsignedLong previous_total_balance) {
+  public static void attestionInclusion(BeaconState state, UnsignedLong previous_total_balance)
+      throws Exception {
     List<Integer> previous_indices = AttestationUtil.get_previous_epoch_attester_indices(state);
```

Comment:
You don't need to prefix the call to `get_previous_epoch_attester_indices ` with `AttestationUtil` since this code is in `AttestationUtil`.

Reply:
This is in EpochProcessor.Util, so it should be necessary.